### DISCUSSION TITLE
Moving between keyframes

### DISCUSSION THREAD

**Discussion by axiomgraph** [friction2d/friction#62] [Ideas]
I think adding an option for moving between keyframes will make editing keyframes easier. and shortcuts too
![move between keyframes](https://github.com/friction2d/friction/assets/68183694/d809cb6c-f4e6-4ff9-a48e-e7de4c1f6fa9)

**Comment by AlexKiryanov** [comment]
Agree, support!

**Comment by rodlie** [comment]
Sorry for the late reply, didn't notice this discussion.

Friction already has (some of) this feature, use `Key_Up` or `Key_Down` on selected object(s) when timeline has focus.

For more shortcuts (not complete) see https://friction.graphics/documentation/shortcuts.html

**Reply by axiomgraph** [reply]
There was a shortcut! Thanks for the help. and I think adding a button inside the user interface may help new users.

**Reply by nuxttux** [reply]
Is it possible to modify the shortcuts?

**Reply by pgilfernandez** [reply]
> Is it possible to modify the shortcuts?

Not yet, as far as I know it is something planned but not implemented yet.
